### PR TITLE
Add troubleshooting for COManage petition stuck in "Approved" State

### PR DESCRIPTION
**Pull request by williamnswanson**
Added section on fixing a CO Person when their CO Petition gets stuck on the "Approved" state to the COManage troubleshooting doc.

### CODE REVIEW

File: docs/operations/comanage-troubleshooting-guide.md [hunk 1]
```diff
@@ -26,6 +26,26 @@ We have seen this occur if a user clicks the confirmation link then closes the t
...
+
+This may happen when the approver has internet issues while attempting to approve a CO Petition.
+
+1.  Navigate to the Petitioner's CO Person page (this can be done directly from the CO Petition under the `Attched Identities` section).
+
+1.  Confirm that the CO Person is missing identifiers of types `UID` and `OSG ID`.
+
+1.  On the CO Person page, confirm that the CO Person's status and Role status are both `Approved`.
+
+1.  On the broken CO Person page, click `Autogenerate Identifiers` on the right side of the page.
```
Comment (brianhlin) [inline docs/operations/comanage-troubleshooting-guide.md:39]: I think steps 2 + 4 here can be combined into one:

"If the CO Persion is missing identifiers..., click `Autogenerate Identifiers`..."

File: docs/operations/comanage-troubleshooting-guide.md [hunk 2]
```diff
@@ -26,6 +26,26 @@ We have seen this occur if a user clicks the confirmation link then closes the t
...
+
+1.  In the `Status` dropdown in the Person Attributes section, change the CO Persons status to `Active` from `Approved`.
+
+1.  Click on the gear icon for the CO Person's Role attribute, then click "Edit", this will bring you to the CO Person Role page
+
+1.  On the CO Person Role page, set the Role Attribute `Status` to `Active` from `Approved`, then click save.
```
Comment (brianhlin) [inline docs/operations/comanage-troubleshooting-guide.md:45]: Similarly, I'd collect these actions together. Maybe a bullet with 3 sub bullets

```
1. If their status is Approved:
    1. Do this
    2. Do this other thing
    3. Then this last thing
```

**Review by brianhlin — APPROVED** [review]
Two suggestions but otherwise LGTM. Pre-approving.

**Comment by williamnswanson** [comment]
Committed the requested changes, but I don't have the write access to merge the PR.

**Pull request merged**